CONTRIBUTING.md

```diff
@@ -13,6 +13,35 @@ Once the python virtual environment is setup, you can run pre-commit hooks using
 pre-commit run --all-files
 ```
 
+## Development dependencies
+
+To install development dependencies, first you need to create a virtual environment.
+The easiest way is by using the [virtualenv](https://pypi.org/project/virtualenv/) package:
```

Comment:
Should we explicitly mention we expect python between 3.9 and 3.11? Or it is it enough to have it in the pyproject.toml

Reply:
I think having it in `pyproject.toml` is enough, we don't have a hard requirement on it ourselves. Besides, when we change it, it's gonna be one less thing for us to worry about.